Framework/Core/test/benchmark_HistogramRegistry.cxx

```diff
@@ -28,9 +28,9 @@ static void BM_HashedNameLookup(benchmark::State& state)
 {
   for (auto _ : state) {
     state.PauseTiming();
-    std::vector<HistogramSpec> specs;
+    std::vector<std::tuple<uint32_t, TH1*>> specs;
     for (auto i = 0; i < state.range(0); ++i) {
-      specs.push_back({(boost::format("histo%1%") % (i + 1)).str().c_str(), (boost::format("Histo %1%") % (i + 1)).str().c_str(), {"TH1F", 100, 0, 1}});
+      specs.push_back(histSpec<TH1F>((boost::format("histo%1%") % (i + 1)).str().c_str(), (boost::format("Histo %1%") % (i + 1)).str().c_str(), "TH1F", {{100, 0, 1}}));
```

Comment:
Please use fmt::format, not boost format. Why do you need the type as a string? Also, why do you return a `tuple<uint32_t, TH1*>`? A std::pair would have been ok, no? Why not just the HistogramSpec as before?

Reply:
Ok, I will adjust the strings and tuple.
If I return histogram spec, I lose the type information for HistogramRegistry constructor. I would need to have HistogramSpec  or HistogramRegistry templated. Now I just pass ready histograms of proper types as TH1*.